src/containers/dashboard/MapInfoDistrict.js

```diff
@@ -16,89 +16,143 @@ class DistrictsMapInfo extends Component {
       ? districtTotals(filteredReportsByDateRange, currentDistrict)
       : null;
     return (
-			<div className="MapInfo_Container">
-				{categoryDisplay === 0 && currentDistrict ? (
-					<React.Fragment>
-						<div className="MapInfo_Section">
-							<div className="color-dark-blue larger-text">District</div>
-							<div className="color-dark-blue medium-text">{currentDistrict.district}</div>
-							<div className="color-light-gray">Date Range</div>
-							<div className="color-dark-blue small-text">
-								{formatDateData(firstReportDate)} - {formatDateData(requestedReport.reportedDate)}
-							</div>
-						</div>
+      <div className="MapInfo_Container">
+        {categoryDisplay === 0 && currentDistrict ? (
+          <React.Fragment>
```

Comment:
I learned a few months ago that you can just use empty tags for this now-- <> </>

Reply:
Ahh that's handy.